### PR TITLE
Fix #116 #117 local variable prevents status refactored for master

### LINKED ISSUE #116: [bug] Status not updated on tray menu
Tested on pre-release bits 1.30.0

There is a regression failure, status on main menu from tray it is not updated according to cluster state. It is always showing `Stopped` 

image (1)

### PR DESCRIPTION
**Pull request by gbraad**
- Issue #116 #117 fix local creation of the status label
- Refactor: remove duplication around statuslabel
- Set correct filemode on LICENSE file (else file will cause conflicts on WSL2)

This aligns master with the 0.9.1 release of the tray binary that does not display status event status due a null pointer exception

### CODE REVIEW

File: CRCTray.cs [hunk 1]
```diff
@@ -77,9 +77,8 @@ private void SetContextMenu()
             cm.AccessibleName = "menu";
 
             // Status Menu
-            ToolStripLabel status = new ToolStripLabel(InitialState); //TODO: actually "Unknown"
-            status.AccessibleName = status.Text;
-            status.TextChanged += Status_TextChanged;
```
Comment (adrianriobo) [inline CRCTray.cs:82]: Do we need to remove this too?  `status.TextChanged += Status_TextChanged;` the fix only removes the local variable

**Comment by gbraad** [comment]
`LICENSE` file was externally added and caused issues with the rebase. Changed file mode as part of the PR

**Comment by gbraad** [comment]
Thw change you.suggeat would not update the accessible text in that case.
In a later commit this is changed to a setstatustest function

**Comment by adrianriobo** [comment]
Ahh yeah was asking for the event remote on text change...but then saw the setstatus function

**Review by adrianriobo — APPROVED** [review]

**Comment by gbraad** [comment]
> Ahh yeah was asking for the event remote on text change...but then saw the setstatus function. I think I deleted the comment before you saw it :) 

Was on phone/email, do harder to see the context it the actual message. If ok, please approve

**Review by adrianriobo — APPROVED** [review]

**Review by adrianriobo — APPROVED** [review]

File: CRCTray.cs [hunk 2]
```diff
@@ -305,24 +299,30 @@ private void ShowDetailedStatusForm(object sender, EventArgs e)
...
         private void DeleteReceived(DeleteResult result)
         {
-            status.Text = InitialState;
+            SetStatusText(InitialState);
+        }
+
+        private void SetStatusText(string state)
+        {
+            status.Text = state;
+            status.AccessibleName = state;
```
Comment (anjannath) [inline CRCTray.cs:325]: 👍🏽

**Comment by anjannath** [comment]
Is it normal to have 755 permission for the LICENSE file?

**Review by anjannath — APPROVED** [review]

**Pull request merged**